### PR TITLE
GitHub sync

### PR DESCRIPTION
**Pull request by nshahan**
A few more updates before publishing. Includes some cleanup of typos and wording from #292.

### CODE REVIEW

File: lib/src/utils/async/simple_stream.dart [hunk 1]
```diff
@@ -403,3 +409,27 @@ class SimpleStreamSubscription<T> implements StreamSubscription<T> {
     throw new UnsupportedError('Not supported.');
   }
 }
+
+/// Provides an interface for both StreamController & and Stream for use with
+/// output events in Angular components.
+///
+/// Reduces the amount of boilerplate needed by removing the need for a getter
+/// that returns the stream for angular.
+class SimpleEmitter<T> extends SimpleStream<T> {
```
Comment (chalin) [inline lib/src/utils/async/simple_stream.dart]: Interesting

File: lib/src/utils/async/simple_stream.dart [hunk 2]
```diff
@@ -403,3 +409,27 @@ class SimpleStreamSubscription<T> implements StreamSubscription<T> {
...
+      bool runInZone = true,
+      SubscriptionChangeListener onListen,
+      SubscriptionChangeListener onCancel})
+      : super.broadcast(
+            isSync: isSync,
+            runInZone: runInZone,
+            onListen: onListen,
+            onCancel: onCancel);
+
+  /// Returns referene to object.
```
Comment (chalin) [inline lib/src/utils/async/simple_stream.dart]: There is a spelling error. How about:

> Returns `this`.

"Returns reference to object" doesn't really tell us which object.

File: lib/src/utils/async/simple_stream.dart [hunk 3]
```diff
@@ -403,3 +409,27 @@ class SimpleStreamSubscription<T> implements StreamSubscription<T> {
...
+  Stream<T> get stream => this;
+
+  /// Returns referene to object.
```
Comment (chalin) [inline lib/src/utils/async/simple_stream.dart]: How about:

> Returns `this`.

**Comment by nshahan** [comment]
Haha, looks like my pubspec update didn't land for some reason. Luckily we should be compatible and we are no longer pinning the release so pub can pull it in if your package depends on it or if you `pub upgrade`.

**Pull request merged**